readme.md

```diff
@@ -84,8 +97,15 @@ $service->loadIdentifier('Authentication.Orm', [
 ]);
 
 // Load the authenticators leaving Basic as the last one.
-$service->loadAuthenticator('Authentication.Session');
-$service->loadAuthenticator('Authentication.Form');
+$service->loadAuthenticator('Authentication.Session', [
+    'sessionKey' => 'Auth'
```

Comment:
You are repeating the default value here. It is not necessary to set this unless you care about what key it is. You should either add that info or remove the config here.